pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -329,6 +329,8 @@ def handle_command(line, args, dryrun=False):
             continue
         if arg == "-lffi":
             continue
+        if arg == "-mpopcnt":
```

Comment:
Maybe let's add a comment that this is a SSE 4.2 instruction, and we currently build without SIMD.

Just so it's easier to know what to re-enable once we want to build with SIMD.

Reply:
Added the comment -